packaging/docker-rh8/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM centos:8
```

Comment:
As noted elsewhere, it at least looks odd to be using a centos 8 base image for what we claim to be our docker rhel 8 image.

Reply:
Changing that (which would be ideal) will probably require some other fixups in this file, especially on the mirrorlist.

Reply:
The image _is_ Centos 8, the same as the image we build for AIO currently is Centos 7 and not RHEL 7. 
RH is for "RedHat" which covers both Centos and RHEL